core/templates/dev/head/pages/state_editor/StateInteractionEditorDirective.js

```diff
@@ -98,10 +98,56 @@ oppia.directive('stateInteractionEditor', [
               interactionCustomizationArgs, false);
           };
 
+          var _updateInteractionPreviewAndAnswerChoices = function() {
```

Comment:
Could you reuse the logic in StateEditorService.getAnswerChoices()? Better than duplicating.

Reply:
done